crates/red_knot_module_resolver/src/typeshed.rs

```diff
@@ -1,22 +1,32 @@
-mod versions;
+use once_cell::sync::Lazy;
 
+use ruff_db::vendored::VendoredFileSystem;
```

Comment:
```suggestion
use ruff_db::vendored::VendoredFileSystem;

```

Reply:
RustRover disagrees with this. I need to use the `self::` prefix to get the desired sorting